src/reporters/LambdaReporter.ts

```diff
@@ -0,0 +1,104 @@
+import { HttpClient } from "@actions/http-client";
+import { throws } from "assert";
+import { CheckOptions } from "../check";
+import { Result } from "../ComplianceChecker";
+import { Product } from "../model/Product";
+import { Reporter } from "./Reporter";
+
+interface ResultRecord {
+  level: string,
+  rule?: string,
+  check?: string,
+  mandatory?: string,
+  result: string,
+  reason?: string
+}
+
+interface RequestBody {
+  repo_owner?: string,
+  repo_name?: string,
+  start_time?: string,
+  end_time?: string,
+  result?: ResultRecord[]
+}
+
+export class LambdaReporter extends Reporter {
+
+  private readonly uri: string;
+  private readonly keyHeader: any;
```

Comment:
<details>
<summary>:warning: <strong>This code is using any type.</strong> (click for details)</summary>

Using the any type defeats the purpose of using TypeScript. When `any` is used, all compiler type checks around that value are ignored.  
  
  
An important improvement would be to use a specific type instead of any or add a justifying comment explaining why any should be used in this code.

**[More details](https://confluence.devfactory.com/x/pzp7HQ)**

## The improvement suggestion in this case
Consider using a specific type instead of any or add a justifying comment explaining why any should be used in this code.   
```ts  
// describe the decision why any is used here
private readonly keyHeader: any;
```
</details>

[//]: # (<a id="1a39cc36-d02a-4933-ba57-8b9dc99dbdbd" data-finding-id="494615207" href="none" codereview-uuid-reserved-tag data-business-id="55ce652a-40cf-48e1-916a-f8d32a3b8762" data-analyzer="js" />)